tools/MakeNiiFilesBIDSCompliant.pl

```diff
@@ -33,12 +33,13 @@ =head1 SYNOPSIS
 =head1 DESCRIPTION
 
 This **BETA** version script will create a BIDS compliant NIfTI file structure of
-the MINC files currently present in the `assembly` directory. If the argument
-`tarchive_id` is specified, only the images from that archive will be processed.
-Otherwise, all files in `assembly` will be included in the BIDS structure,
-while looping though all the 'tarchive_id`'s in the 'tarchive` table.
+the MINC files currently present in the C<assembly> directory. If the argument
+C<tarchive_id> is specified, only the images from that archive will be
+processed. Otherwise, all files in C<assembly> will be included in the BIDS
+structure, while looping though all the C<tarchive_id>'s in the C<tarchive>
+table.
```

Comment:
Do these "C< ... >" tag need to be here?

Reply:
yeah, it is to put that in code format so that it automatically converts into `\`` in the .md file